### PR TITLE
*: add `exists_in_docs_cn: false` and fix invalid links

### PR DESCRIPTION
**Pull request by CaitinChen**
Related PR: https://github.com/pingcap/docs/pull/1182
@dcalvin @YiniXu9506 PTAL

### CODE REVIEW

File: dev/reference/garbage-collection.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 title: TiDB Garbage Collection (GC)
 summary: Use Garbage Collection (GC) to clear the obsolete data of TiDB.
 category: reference
-aliases: ['/docs/op-guide/gc/'] 
+aliases: ['/docs/op-guide/gc/']
+exists_in_docs_cn: false
```
Comment (lilin90) [inline dev/reference/garbage-collection.md]: This file does exist in docs-cn, just the existing file name is not consistent with that in docs. I've updated it in https://github.com/pingcap/docs-cn/pull/1401.

File: dev/reference/supported-clients.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
 title: Connectors and APIs 
 summary: Learn about the connectors and APIs.
 category: reference
-aliases: ['/docs/sql/connection-and-APIs/'] 
+aliases: ['/docs/sql/connection-and-APIs/']
```
Comment (lilin90) [inline dev/reference/supported-clients.md]: ditto

**Comment by lilin90** [comment]
@CaitinChen Please address comments. Thanks!

**Comment by CaitinChen** [comment]
@lilin90 Thanks. PTAL again.

**Pull request merged**